src/Tooltip/Tooltip.stories.tsx

```diff
@@ -321,16 +321,41 @@ const VisualTestTemplate: StoryFn<typeof Tooltip> = (args) => {
 const InteractiveExampleTemplate: StoryFn<typeof Tooltip> = (args) => {
   // Placed button in center position of the screen to test all tooltip placements
   return (
-    <Tooltip {...args}>
-      <Button sx={{
-        position: 'absolute',
-        left: '50%',
-        top: '50%',
-      }}
+    <>
+      <Tooltip {...args}>
+        <Button sx={{
+          position: 'absolute',
+          left: '50%'
+        }}
+        >
+          Button
+        </Button>
+      </Tooltip>
+      <Tooltip
+        {...args}
+        maxwidth={327}
+        tooltipsize="Small"
+        placement="bottom-start"
```

Comment:
```suggestion
```

You can omit this now. The args will be passed from Storybook control in Line 335. @ombhagat05

Reply:
Done